app/assets/stylesheets/application.scss

```diff
@@ -18,6 +18,8 @@ body {
   height: 100%;
 }
 
+.clearfix {clear:both;}
```

Comment:
I don't understand why this is needed, but something has removed this from clearfix in our css, so cleafix didn't do its job :cry: